tests/nurse_consent_no_response.spec.ts

```diff
@@ -44,7 +44,7 @@ async function when_i_select_a_child_with_no_consent_response() {
 }
 
 async function when_i_click_get_consent() {
-  await p.getByRole("link", { name: "Get consent" }).click();
```

Comment:
Why is this change not like the others?

Reply:
This one is actually initially a button and then it's a link. It's a button when it's creating a consent, and a link when it's editing one. One alternative is to create two different `when_i_click_*` methods, which would also work

Reply:
If there's a reason for it then it's probably ok to leave. There's always the chance the test breaks if "Get consent" shows up elsewhere, but we can wait for that calamity to hit before we act on it.